packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuItem.Props.ts

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import { IIconProps } from '../../Icon';
+import { IContextualMenuProps } from './ContextualMenu.Props';
+
+export interface IContextualMenuItem {
+  /**
+   * Unique id to identify the item
+   */
+  key: string;
+
+  /**
+   * Text description for the menu item to display
+   */
+  name?: string;
+
+  /**
+   * Type of item in the menu. Includes normal, divider or header.
+   */
+  menuItemType?: ContextualMenuItemType;
```

Comment:
This is probably going to be problematic since this concept doesn't exist for a button. I don't view headers or dividers as menu items really. Instead, we should probably have a concept of a menu section that can have dividers after them.